### ISSUE TITLE
Make IiifPrint work with Valkyrie

### ISSUE THREAD

**Issue by ShanaLMoore** [scientist-softserv/iiif_print#312]

**Comment by ShanaLMoore** [comment]
restrict hyrax version that iiif print support.

**Comment by jeremyf** [comment]
> restrict hyrax version that iiif print support.

I don't think we want to overly restrict; I'd rather spend some time allowing for configurations to address the different versions we're supporting.

Consider that we have some IIIF Print applications on Hyrax 2.9.6 and some on what will be Hyrax 5.  With https://github.com/scientist-softserv/iiif_print/pull/313 we have a pattern for how we can work with both.

**Comment by jillpe** [comment]
Blocked: Waiting to be updated in hyku
